### DISCUSSION TITLE
Alpine doesn't work for me on IOS (Safari and chrome)

### DISCUSSION THREAD

**Discussion by RoylerMarichal** [alpinejs/alpine#2805] [1. Help]
I have a project with the TALL stack without any extra dependencies, it runs perfectly on Android and Windows, on chrome and edge. But in IOS, Safari or chrome alpine doesn't work for me at all... Any suggestions?

**In app layout**
`  <script src="//unpkg.com/alpinejs" defer></script> `
`  @livewireStyles `

 -- Content 

 `@livewireScripts`

**Ej. Menu Alpine**
`<div x-data="{menu:false}">`
...
  ` <button type="button" x-on:click="menu=!menu" >`
...
`  <div x-cloak x-show="menu" >`
...

The website is [https://yavoycuba.com](https://yavoycuba.com)

**Comment by SimoTod** [comment]
See https://github.com/alpinejs/alpine/discussions/2420, it's probably the same issue

**Reply by RoylerMarichal** [reply]
I tried with v2 too... I have a single function for each component, as simple as possible, look at the yavoycuba.com website and try to open the menu from an IOS
